### DISCUSSION TITLE
Dart version managment

### DISCUSSION THREAD

**Discussion by blondie63** [leoafarias/fvm#532] [Q&A] [answered]
fvm is very good to have multiple flutter version but for Dart?

for example now i need to use Flutter 3.0.2 and Dart 2.19.2 because i'm using a package still not ready for Dart 3.x
how can i do ?

Thanks for help

**Comment by PackRuble** [comment]
As a workaround you can install required flutter-beta versions... And use dart sdk from `.fvm\flutter_sdk\bin\cache\dart-sdk` folder

**Comment by leoafarias** [comment]
@blondie63 FVM 3 will have a bit better support for the Dart SDK, so I am curious about your experience with it. It is not meant to manage Dart SDK directly, but better support the Dart SDK within the Flutter SDKs

**Reply by blondie63** [reply]
@leoafarias now i'm using only stable versions of flutter and dart so i don't have the problem anymore..

**Comment by deandreamatias** [comment]
If have a nice support to Dart with FVM 3.0, can close this discussion.
Maybe can add some documentation and keyword on readme to reference to Dart Version Management concept

**Reply by leoafarias** [reply, marked as answer]
Correct FVM now does constraint checking. It also does better checks for dependencies and dart versions of packages. I do want to be able to install a Flutter version based on the Dart SDK constraint, and that is on my wish list ;)
